### ISSUE TITLE
"failed to fetch user profile" after the user just registered, but user created successfully

### ISSUE THREAD

**Issue by retorquere** [auth0/passport-auth0#41]
After registration, I get:

```
Error: failed to fetch user profile
    at .../node_modules/passport-auth0/lib/index.js:128:28
    at passBackControl (.../node_modules/oauth/lib/oauth2.js:132:9)
    at IncomingMessage.<anonymous> (/Users/emile/github/cvrm/node_modules/oauth/lib/oauth2.js:157:7)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```
but subsequent attempts log in just fine

**Comment by retorquere** [comment]
The error traces back to https://github.com/auth0/passport-auth0/blob/master/lib/index.js#L128 -- I allow creation of new users but have them blocked by default by a rule, and that yields an "user is blocked" error. I could deal with that with an express error handler, but that line doesn't actually pass the inner error in a way that I can access it in the error handler; perhaps it would be better to change that line to something like

```
if (err) {
  var error = new Error('failed to fetch user profile')
  error.inner = err
  return done(error);
}
```

**Comment by joshcanhelp** [comment]
Duplicate of #20
